examples-of-custom-resources/egress-mtls/egress-mtls.yaml

```
@@ -0,0 +1,12 @@
+apiVersion: k8s.nginx.org/v1alpha1
+kind: Policy
+metadata:
+  name: egress-mtls-policy
+spec:
+  egressMTLS:
+    tlsSecret: egress-mtls-secret
+    trustedCertSecret: egress-trusted-ca-secret
+    verifyServer: on
+    verifyDepth: 2
+    serverName: on
+    sslName: "secure-app.example.com"
```

I don't think we need `"` for hosts
```suggestion
    sslName: secure-app.example.com
```